### PR TITLE
Remove debug print from apphook_reload

### PR DESCRIPTION
**Pull request by yakky**
## Description

Port changes in #6791 as aggreed on the issue

## Related resources

* #6791 

## Checklist

* [x] I have opened this pull request against ``develop``
* [x] I have updated the **CHANGELOG.rst**
* [ ] I have added or modified the tests when changing logic

### CODE REVIEW

**Review by FinalAngel — APPROVED** [review]

**Comment by yakky** [comment]
@FinalAngel @Aiky30 could you restart the failed travis task? failure seem more random than related to this, and apparentyl I can't restart tasks on travis (probably due to the org change)

**Comment by Aiky30** [comment]
@yakky @FinalAngel I don't have the access to rerun this test I'm afraid. I don't think I was ever able to in Travis, I could in Circle CI.

**Comment by FinalAngel** [comment]
@yakky @Aiky30 you may need to re-trigger the build. They should no go through the django-cms organisation (which should allow you to restart the builds)

**Comment by yakky** [comment]
@FinalAngel forced a push to retrigger the build. I confirm that I can now cancel / retry builds on travis

**Review by Aiky30 — APPROVED** [review]
Ideally any change will have a changelog. Willing to let this go here but won't in future.

**Comment by Aiky30** [comment]
Needs bringing up to date and then it's ready to come in.

**Comment by yakky** [comment]
@Aiky30 sorry I missed your comment

Changelog entry added

**Review by Aiky30 — APPROVED** [review]

**Comment by yakky** [comment]
@Aiky30 conflict fixed

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,6 +7,7 @@ Unreleased
 ==================
 
 * Fixed builds on RTD
+* Remove debug print from appok_reload
```
Comment (merwok) [inline CHANGELOG.rst:10]:
```suggestion
* Remove debug print from apphook_reload
```

Comment (yakky) [reply CHANGELOG.rst]: :bow:

**Pull request merged**